bgpd/bgp_evpn.c

```diff
@@ -2119,10 +2119,11 @@ static int update_evpn_route(struct bgp *bgp, struct bgpevpn *vpn,
 		char buf3[ESI_STR_LEN];
 
 		zlog_debug(
-			"VRF %s vni %u type-2 route evp %pFX RMAC %pEA nexthop %pI4 esi %s",
+			"VRF %s vni %u type-%u route evp %pFX RMAC %pEA nexthop %pI4 esi %s",
 			vpn->bgp_vrf ? vrf_id_to_name(vpn->bgp_vrf->vrf_id)
```

Comment:
Yes. But `bgp_vrf` is non-exist in this case, so doesn't work.

Reply:
Sorry, it's an unrelated change :smile: Just spotted it now :shrug:

Reply:
None means default VRF or what actually in this EVPN context?

Reply:
Here "None" means "vpn->bgp_vrf" has no relation to one "vrf" in `bgpd`. Actually "vpn->bgp_vrf" is not created at that time, only "vrf" is created.

Reply:
In what situation can this log be triggered (just to understand it clearly)?

Reply:
1. In FRR, set evpn in default vrf: (only set `router bgp X`, not set `router bgp X vrf vrf1`)
```
 address-family l2vpn evpn
  neighbor 88.88.88.1 activate
  advertise-all-vni
```
2. Create SVI in shell: "vlan100" (master to "vrf1") and `vni 100`. So, zebra will notify bgpd this l2vni `vni 100` information.
3. `bgpd` received it, but it can't find `evpn->bgp_vrf` by `vni100`, then trigger this log.